Jellyfin.Api/Helpers/TranscodingJobHelper.cs

```diff
@@ -350,5 +361,30 @@ private void DeleteHlsPartialStreamFiles(string outputFilePath)
                 throw new AggregateException("Error deleting HLS files", exs);
             }
         }
+
+        private void OnPlaybackProgress(object sender, PlaybackProgressEventArgs e)
+        {
+            if (!string.IsNullOrWhiteSpace(e.PlaySessionId))
+            {
+                PingTranscodingJob(e.PlaySessionId, e.IsPaused);
+            }
+        }
+
+        /// <summary>
+        /// Deletes the encoded media cache.
+        /// </summary>
+        private void DeleteEncodedMediaCache()
```

Comment:
This is unused?

Reply:
I believe this PR is to bring the remaining functions over to Jellyfin.Api that many other controllers rely on

Reply:
This function was called on server startup but I haven't found a way to call it yet. 
The main goal of this PR was to bring the remaining functions to Jellyfin.Api. But I think I created more confusion than I helped.